cli/src/api.ts

```diff
@@ -16,9 +16,6 @@ export type CheckAuthResult = {
 export type SpawnRequestBody = {
   env?: Record<string, string>; // env vars always map strings to strings
   grace_period_seconds?: number;
-  port?: number;
```

Comment:
the port is gone too?

Reply:
yeah - for now we're going to see if people can get by only using 8080. We can always add it back if people end up really needing to use custom ports